src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -44,15 +46,20 @@ const TernaryRadios = ({
     if (!formValue) {
       setFormError(true);
       focusElement(radioRef.current);
-    } else {
-      if (valueHasChanged) {
-        // Remove answers from the Redux store if the display path ahead has changed
-        cleanUpAnswers(formResponses, updateCleanedFormStore, shortName);
-      }
 
+      return;
```

Comment:
This function was a bit messy. I broke it up into a few separate `if` blocks, and this one required a `return` so it didn't try to navigate forward if there is a form error.